### DISCUSSION TITLE
Can i join the team?

### DISCUSSION THREAD

**Discussion by paulowesll** [ROM-builders/temporary#17736] [General]
device tree:
https://github.com/paulowesll/android_device_samsung_m52xq

common device tree:
https://github.com/paulowesll/android_device_samsung_sm7325-common

vendor tree:
https://github.com/paulowesll/android_vendor_samsung_sm7325-common

kernel tree:
https://github.com/paulowesll/android_kernel_samsung_sm7325

telegram:
https://t.me/paulowesll

lasted successfully build completed:
https://cirrus-ci.com/build/5684658256478208

(more than 10 builds have already been successfully compiled, all booted)

**Comment by Apon77** [comment]
telegram and github username / name mismatching! can't be added

**Reply by paulowesll** [reply]
I've updated my telegram username to the same as my github, i edited it, if you can check again please.
